frontend/src/components/app/app.tsx

```diff
@@ -17,6 +18,7 @@ const App: React.FC = () => (
       <Route path={AppRoute.SIGN_UP} component={SignUp} exact />
       <Route path={AppRoute.ANY} component={NotFound} />
     </Switch>
+    <HomePage/>
```

Comment:
Home page should be accessible by some route

Reply:
added home page to routing